types/office-js-preview/index.d.ts

```diff
@@ -5104,6 +5104,25 @@ declare namespace Office {
          */
         type: EventType;
     }
+    /**
+     * Provides information about the message from the parent page that raised the DialogParentMessageReceived event.
+     * 
+     * @beta
+     * 
+     * To add an event handler for the DialogParentMessageReceived event, use the addHandlerAsync method of the 
+     * {@link Office.UI} object.
+     * 
+     */
+    interface DialogParentMessageReceivedEventArgs {
+        /**
+         * Gets the content of the message sent from the parent page, which can be any data that can be stringified.
```

Comment:
The double "can" is little repetitive.
```suggestion
         * Gets the content of the message sent from the parent page, which can be any stringified data.
```